docs/UserGuide.md

```diff
@@ -72,74 +72,87 @@ Shows a message explaning how to access the help page.
 Format: `help`
 
 
-### Adding a person: `add`
+### Adding a student to contact book: `add`
 
-Adds a person to the address book.
+Adds a student to the address book.
```

Comment:
Should we should standardise to use the term `contact book` instead of `address book` ?